src/django_components/commands/ext_list.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any
+
+from django_components.extension import extensions
+from django_components.util.command import CommandArg, ComponentCommand
+
+
+class ExtListCommand(ComponentCommand):
```

Comment:
Definition for `python manage.py components ext list`, which lists the names of all installed extensions. Mainly for debugging